### DISCUSSION TITLE
Unable to change the size of font on plots in 5.x

### DISCUSSION THREAD

**Discussion by smk78** [SasView/sasview#2377] [General]
User RebekaB notes that it is not possible to change the size of font on plots in 5.0.4.

Investigations by @butlerpd and @smk78 show:

**In 4.2.2.**
You _could_ change the font, style, size and weight of the text on the axis labels.
![image](https://user-images.githubusercontent.com/10938679/136262962-21c72fa1-8f9c-41c6-a5a8-c1583283ebe2.png)
But there was no way to change any properties of the legend text, only toggle it on/off.

**In 5.0.4.**
There is also no way to change any properties of the legend, only toggle it on/off again. So the behaviour of 5.x is the same as 4.x in this regard.
![image](https://user-images.githubusercontent.com/10938679/136263611-83af7148-9e8a-4281-b7b2-a590069f86bc.png)
However, unlike 4.x, in 5.x it is not possible to change the properties of the axis labels. This, at least, needs rectifying for feature parity.
![image](https://user-images.githubusercontent.com/10938679/136263863-ac3ae917-c408-4ebd-831d-707b6dac6da9.png)

But in rectifying the behaviour of 5.x, it would also be good to provide control of the properties of the legend text.

**Comment by rozyczko** [comment]
Added a widget allowing customisation of the MPL legends.
Added the trigger for it in the right-click plot menu

**Comment by lucas-wilkins** [comment]
Close this? @rozyczko

**Comment by rozyczko** [comment]
Sure but I'd wait until it's reviewed by @smk78 and merged

**Comment by smk78** [comment]
See https://github.com/SasView/sasview/pull/2096#issuecomment-1159274615
